package-res/resources/web/prompting/PromptPanel.js

```diff
@@ -354,6 +354,10 @@ define(['cdf/lib/Base', 'cdf/Logger', 'dojo/number', 'dojo/i18n', 'common-ui/uti
               return paramValue.toUpperCase() != paramSelectedValue.toUpperCase();
             case "java.sql.Date": // Set time to zero to eliminate its influence on the days comparison
               return (new Date(paramValue).setHours(0,0,0,0)) != (new Date(paramSelectedValue).setHours(0,0,0,0));
+            case "[Ljava.lang.String;": // Comparing String arrays
```

Comment:
thanks @tmorgner for the info, will change this to handle all array types then :)